src/color_laws.jl

```diff
@@ -137,3 +137,39 @@ function cal00_invum(x::Real, Rv::Real)
     return 1.0 + k / Rv
 
 end
+
+
+"""
+    gcc09(λ::Real, Rv=3.1)
+    gcc09(λ::Quantity, Rv=3.1)
+
+Gordon, Cartledge, & Clayton (2009) dust law.
+
+If `λ` is a `Unitful.Quantity` it will be automatically converted to Å and the
+returned value will be `UnitfulAstro.mag`.
+"""
+function gcc09(λ::Real, Rv = 3.1)
+    x = aa_to_invum(λ)
+    return gcc09_invum(x, Rv)
+end
+
+gcc09(λ::Quantity, Rv::Real = 3.1) = gcc09(ustrip(u"angstrom", λ), Rv) * u"mag"
+
+function gcc09_invum(x::Real, Rv::Real)
+
+    a = zeros(typeof(Rv))
+    b = zeros(typeof(Rv))
```

Comment:
`zeros` generates an array, I think you wanted to use `zero` instead.  Actually, I don't think these are necessary at all?

Reply:
Correct! Thank you!